ExtraDry/ExtraDry.Server/Models/SortStabilization.cs

```diff
@@ -0,0 +1,27 @@
+﻿namespace ExtraDry.Server
+{
+    /// <summary>
+    /// Defines the behaviour of ExtraDry when providing a sort method to an underlying data store.
+    /// </summary>
+    public enum SortStabilization
+    {
+        /// <summary>
+        ///  The default behaviour, this will use a secondary property on sorts to provide consistent results
+        /// </summary>
+        AlwaysAddKey = 0,
```

Comment:
Is there a benefit in explicitly assigning values?  E.g. they're not persisted to a database so order changes won't cause serialization issues...

Reply:
No, habit. As long as the default is 0, then the rest don't matter